app/src/main/java/com/devshish/internship/presentation/ui/service/client/MediaBrowserClient.kt

```diff
@@ -0,0 +1,54 @@
+package com.devshish.internship.presentation.ui.service.client
+
+import android.content.ComponentName
+import android.content.Context
+import android.support.v4.media.MediaBrowserCompat
+import android.support.v4.media.session.MediaControllerCompat
+import android.support.v4.media.session.PlaybackStateCompat
+import androidx.core.net.toUri
+import com.devshish.internship.presentation.ui.service.server.MediaBrowserService
+
+class MediaBrowserClient(context: Context) {
+
+    private lateinit var mediaBrowser: MediaBrowserCompat
+    private lateinit var mediaController: MediaControllerCompat
+
+    private val controllerCallback = MediaControllerCallback()
+
+    init {
+        mediaBrowser = MediaBrowserCompat(
+            context,
+            ComponentName(context, MediaBrowserService::class.java),
+            object : MediaBrowserCompat.ConnectionCallback() {
+                override fun onConnected() {
+                    super.onConnected()
+                    mediaController =
+                        MediaControllerCompat(context, mediaBrowser.sessionToken).apply {
+                            registerCallback(controllerCallback)
+                            transportControls.playFromUri(
```

Comment:
When you add some code that is needed only in current stage of development - leave todo comment to remove this